packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -241,10 +251,43 @@ class DeepLinksController extends DisposableController
         }
       },
     );
+  }
 
+  Future<void> loadIosLinks() async {
+    final iosBuildOptions = selectedProject.value!.iosBuildOptions;
+    final configuration =
+        iosBuildOptions.configurations[selectedIosConfigurationIndex.value];
+    await ga.timeAsync(
+      gac.deeplink,
+      gac.AnalyzeFlutterProject.loadIosLinks.name,
+      asyncOperation: () async {
+        final UniversalLinkSettings result;
+        try {
+          result = await server.requestIosUniversalLinkSettings(
+            selectedProject.value!.path,
+            configuration: configuration,
+            target: iosBuildOptions.targets[0],
```

Comment:
is this guaranteed to no throw and index out of bounds exception?

Reply:
Yes, you're right, If the project just have no ios build options set up, there should be some error handlings.

I will merge https://github.com/flutter/devtools/pull/7880 first and update this pr to use the target from the dropdown menu and add error handling for project without  ios build options set up